action_tutorials/action_tutorials_actions/package.xml

```diff
@@ -1,25 +1,23 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>action_tutorials</name>
+  <name>action_tutorials_actions</name>
   <version>0.7.6</version>
-  <description>Action tutorial code examples</description>
+  <description>Action tutorials action</description>
   <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
   <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
-
+  
   <depend>action_msgs</depend>
 
-  <exec_depend>rclpy</exec_depend>
-  <exec_depend>rosidl_default_runtime</exec_depend>
-
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
 
+  <exec_depend>rosidl_default_runtime</exec_depend>
```

Comment:
nit: I would put `<exec_depend>` tag before the `<test_depend>` tags as done in other packages.